src/Components/Home/ArticleList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "../ArticleCard";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-app-ftk2.onrender.com/api/articles")
```

Comment:
this could be made a lot cleaner, more reusable and maintainable if you were to extract all of your api calls out into a separate file and took a look at axios.create (check the axios notes out)

Reply:
If you were the user, what would you expect to see as you waiting for the articles to be fetched?